packages/spatial/src/renderer/materials/materialFunctions.ts

```diff
@@ -197,6 +197,13 @@ export const assignMaterial = (entity: Entity, materialEntity: Entity) => {
   materialStateComponent.instances.set([...materialStateComponent.instances.value, entity])
 }
 
+export const removeMaterialEntity = (materialEntity: Entity) => {
```

Comment:
This doesn't need to be its own function, let's just use removeEntity(materialEntity). Material entities are handled through queries for the material state component so they're gaurunteed to have this component.

Reply:
implemented